src/XML/src/FMI3/fmi3_xml_parser.c

```diff
@@ -891,6 +891,7 @@ int fmi3_xml_set_attr_array(fmi3_xml_parser_context_t *context, fmi3_xml_elm_enu
 
     /* write all attributes to array of correct type */
     if (fmi3_xml_str_to_array_floatXX(context, str, arrPtr, arrSize, primType)) {
+        fmi3_xml_parse_attr_error(context, elmID, attrID, str);
```

Comment:
@PeterMeisrimelModelon
I removed this when cleaning up error messages during review. I realized it would be still good to still have it. Reasoning is that you should give an error message that contains the context of the failure. That context is {elmID, attrID, str}, which are not passed to fmi3_xml_str_to_array_floatXX.